DESCRIPTION

```diff
@@ -23,3 +23,6 @@ Suggests:
     deSolve,
     testthat
 RoxygenNote: 6.1.1
+Remotes:
+    ropensci/jsonvalidate,
```

Comment:
why do we prefer ropensci over CRAN's version?

Reply:
This is way ahead of CRAN, but in order to get the package updated on CRAN I have to add some backward compatibility for another CRAN package that the updates break